widgetastic_manageiq/__init__.py

```diff
@@ -2614,14 +2614,18 @@ def get_all(self, surf_pages=False):
                                 for el in self._current_page_elements])
             return entities
 
-    def get_entity(self, surf_pages=False, **keys):
+    def get_entity(self, surf_pages=False, use_search=False, **keys):
```

Comment:
**REQUIRED**
Please add `use_search` argument to the docstring.

Reply:
done